app/v2/controllers/V2MovementsController.scala

```diff
@@ -312,7 +362,7 @@ class V2MovementsControllerImpl @Inject() (
             } else stringToByteStringStream(messageSummary.body.get.value)
```

Comment:
I think this will get failed in case of we call this for empty movement(empty movement is going to have any body)?

Reply:
Yup, that's true.